docs/local-testing.md

```diff
@@ -4,258 +4,212 @@ In this walkthrough, you will learn how to use the kCTF infrastructure.
 
-If you have the following type of error:
-```
-[E][2020-01-31T20:16:39+0000][1] bool subproc::runChild(nsjconf_t*, int, int, int)():459 nsjail tried to use the CLONE_NEWCGROUP clone flag, which is supported under kernel versions >= 4.6 only. Try disabling this flag: Operation not permitted
-[E][2020-01-31T20:16:39+0000][1] bool subproc::runChild(nsjconf_t*, int, int, int)():464 clone(flags=CLONE_NEWNS|CLONE_NEWCGROUP|CLONE_NEWUTS|CLONE_NEWIPC|CLONE_NEWUSER|CLONE_NEWPID|CLONE_NEWNET|SIGCHLD) failed. You probably need root privileges if your system doesn't support CLONE_NEWUSER. Alternatively, you might want to recompile your kernel with support for namespaces or check the current value of the kernel.unprivileged_userns_clone sysctl: Operation not permitted
-```
-This probably means that unprivileged user namespaces are not enabled. You can fix this by running:
-```
-(echo 1 | sudo tee /proc/sys/kernel/unprivileged_userns_clone) || (echo 'kernel.unprivileged_userns_clone=1' | sudo tee /etc/sysctl.d/00-local-userns.conf) 2>&1
-sudo service procps restart
-```
-Then try connecting through netcat again.
+NAME          READY   UP-TO-DATE   AVAILABLE   AGE     CONTAINERS              IMAGES                                                                                                                                                              SELECTOR
+chal-sample   1/1     1            1           5m49s   challenge,healthcheck   kind/challenge:6eba2,kind/healthcheck:dcf4   app=chal-sample
 
-### Errors in Windows Subsystem for Linux
+= EXTERNAL SERVICES =
 
-#### WSL1
+Challenge external status
+This shows you if the challenge is exposed externally.
 
-The only **supported** method (see below for unsupported options) for testing on WSL1 is through a local Kubernetes cluster, and this is only possible for users with a Pro/Edu/Enterprise version of Windows 10. This is because [Docker for Windows](https://docs.docker.com/docker-for-windows/) requires [Hyper-V support](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/about/).
+SERVICES:
+NAME          TYPE       EXTERNAL-IP   PORT   DNS
+chal-sample   NodePort   <none>        1337   <none>
 
-You must [download Docker for Windows](https://download.docker.com/win/stable/Docker%20Desktop%20Installer.exe) (1GB in size). It already includes Kubernetes, but the cluster has to be manually started in the UI. Go to *Settings > Kubernetes* and click on **Enable Kubernetes**.
+Ingresses:
+No resources found in default namespace.
 
-Open WSL 1 and enter:
-```
-echo -e '#!/bin/bash\n$(basename $0).exe $@' | tee  $HOME/.local/bin/docker $HOME/.local/bin/kubectl
-chmod +x $HOME/.local/bin/docker $HOME/.local/bin/kubectl
 ```
 
-Make sure you are running the Windows version of Docker and kubectl:
-<!-- {% raw  %} -->
+### Reading logs
+To troubleshoot unhealthy challenges, we need to read the logs of the failing healthcheck. We need to do that using `kubectl`.
```

Comment:
why not `kctf chal debug logs --container healthcheck` ?

Reply:
it doesn't work. #262

Reply:
I see, then let's fix #262 instead